frida_mode/README.md

```diff
@@ -1,38 +1,40 @@
+a small harness around their target code of interest. FRIDA mode instead takes a
+different approach to avoid these limitations. In FRIDA mode, binary programs
+are instrumented, similarly to QEMU mode.
 
-## Current Progress
+## Current progress
 
-As FRIDA mode is new, it is missing a lot of features. The design is such that it
-should be possible to add these features in a similar manner to QEMU mode and
+As FRIDA mode is new, it is missing a lot of features. The design is such that
+it should be possible to add these features in a similar manner to QEMU mode and
 perhaps leverage some of its design and implementation.
 
-  | Feature/Instrumentation  | frida-mode | Notes                                        |
-  | -------------------------|:----------:|:--------------------------------------------:|
-  | NeverZero                |     x      |                                              |
-  | Persistent Mode          |     x      | (x86/x64/aarch64 only)                       |
-  | LAF-Intel / CompCov      |     -      | (CMPLOG is better 90% of the time)           |
-  | CMPLOG                   |     x      | (x86/x64/aarch64 only)                       |
-  | Selective Instrumentation|     x      |                                              |
-  | Non-Colliding Coverage   |     -      | (Not possible in binary-only instrumentation |
-  | Ngram prev_loc Coverage  |     -      |                                              |
-  | Context Coverage         |     -      |                                              |
-  | Auto Dictionary          |     -      |                                              |
-  | Snapshot LKM Support     |     -      |                                              |
-  | In-Memory Test Cases     |     x      | (x86/x64/aarch64 only)                       |
+| Feature/Instrumentation  | frida-mode | Notes                                        |
+| -------------------------|:----------:|:--------------------------------------------:|
+| NeverZero                |     x      |                                              |
+| Persistent Mode          |     x      | (x86/x64/aarch64 only)                       |
+| LAF-Intel / CompCov      |     -      | (CMPLOG is better 90% of the time)           |
+| CMPLOG                   |     x      | (x86/x64/aarch64 only)                       |
+| Selective Instrumentation|     x      |                                              |
+| Non-Colliding Coverage   |     -      | (not possible in binary-only instrumentation) |
+| Ngram prev_loc Coverage  |     -      |                                              |
+| Context Coverage         |     -      |                                              |
+| Auto Dictionary          |     -      |                                              |
+| Snapshot LKM Support     |     -      |                                              |
+| In-Memory Test Cases     |     x      | (x86/x64/aarch64 only)                       |
```

Comment:
Does this table have a particular order? Can I sort alphabetically (here and in other places where the feature table is used)?

Reply:
it is ok as is, same for the other tables, no need :)